Package.swift

```diff
@@ -710,7 +709,10 @@ let package = Package(
       name: "FirebaseFunctionsSwiftUnit",
       dependencies: ["FirebaseFunctionsSwift",
                      "SharedTestUtilities"],
-      path: "FirebaseFunctionsSwift/Tests/Unit"
+      path: "FirebaseFunctionsSwift/Tests/Unit",
+      cSettings: [
+        .headerSearchPath("../../../"),
```

Comment:
I'm curious, what's this needed for?

Reply:
Here's the build error without it:
<img width="900" alt="Screen Shot 2022-02-07 at 8 08 47 AM" src="https://user-images.githubusercontent.com/73870/152837120-8f009ad3-5fb8-428f-863d-0c17e1f70f21.png">